tests/js/HappinessLintBearTest.py

```diff
@@ -0,0 +1,28 @@
+from bears.js.HappinessLintBear import HappinessLintBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+var x = 2;
+console.log(x);
+var message = 'hello, ' + 'world' + '!';
+console.log(message);
+var list = [1, 2, 3, 4];
+if (6 > 7 !== true) console.log('done');
+console.log(list);
+window.alert('hi');
+""".splitlines(keepends=True)
+
+bad_file = """
+function hello (name) {
+ console.log('hi', name)
+}
+if (options.quiet !== true)
+  console.log('done')
+var x=2
+var message = 'hello, '+name+'!'
+""".splitlines(keepends=True)
```

Comment:
Are these tests minimal? See if it is possible to reduce the size of these test cases.